WebUI/lims/views.py

```diff
@@ -242,6 +278,8 @@ def studyNotes(request,studyId,pageNum):
         # Only when endtime and runningtime exist, we process the data
         if rowsList[ndx][3]:
             rowsList[ndx][3] = rowsList[ndx][3].strftime(DATEFORMAT)
+    if len(rowsList) == 0:
+        tableNeed = False
```

Comment:
More of a style preference, but another way of writing this that is common would be:

tableNeed = len(rowsList) != 0